bazel/repository_locations.bzl

```diff
@@ -584,7 +584,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         sha256 = "4be3d92add3e3fad56d09a68b6f3f18367b086a3442275e046e84b3c2bb94d43",
         strip_prefix = "grpc-httpjson-transcoding-{version}",
         urls = ["https://github.com/grpc-ecosystem/grpc-httpjson-transcoding/archive/{version}.tar.gz"],
-        use_category = ["dataplane_ext"],
+        use_category = ["controlplane", "dataplane_core", "dataplane_ext"],
```

Comment:
@moderation since this is a major core dataplane dependency addition, we'd definitely benefit from you double checking on this and https://github.com/envoyproxy/envoy/issues/15304. Thanks.

Reply:
I've updated https://github.com/envoyproxy/envoy/issues/15304#issuecomment-791177043 with some details. To be in compliance with the policy for core deps inclusion the project `MUST` categories missing are CI and Security Policy. @nareddyt is working on CI. Setting up a security policy should also not be too onerous.